Bindings/Java/Matlab/Hopper_Device/RunHopperWithDevice_answers.m

```diff
@@ -78,7 +78,7 @@
 % (Done for you) Configure the device to wrap over the patella.
 if hopper.hasComponent('device_active') || hopper.hasComponent('device_passive') 
     cable = PathActuator.safeDownCast(device.updComponent('cableAtoBactive'));
-    patellaPath = 'thigh/patellaFrame/patella';
+    patellaPath = 'bodyset/thigh/patellaFrame/wrapobjectset/patella';
     wrapObject = WrapCylinder.safeDownCast(hopper.updComponent(patellaPath));
```

Comment:
This maybe late but wrapobjectset feels just an extraneous level, similar to pathpointset (that I reverted to regular rather than ModelComponentSet), should we try that in a separate issue, or is there a use case where it's needed?